GoMeet/src/main/webapp/js/manual-time-proposal.js

```diff
@@ -0,0 +1,161 @@
+const MAX_TIME_INPUTS = 5; // Allow up to 5 proposed times
+var timeInputs = 1; // Current number of time inputs on page
+const MEETING_TIME_NAME_PREFIX = 'meeting-time-'; // The prefix of the sessionStorage key name for meeting times
+var timeNameSuffix = 1; // The suffix appended to MEETING_TIME_NAME_PREFIX to make the name unique
+const MAX_TIME_INPUT_REACHED_ALERT = 'Maximum number of proposed times: ' + MAX_TIME_INPUTS.toString();
+const TIME_INPUT_FIELD_INDEX = 1; // The index of the <input> field with its parent div
+var enteredTimes = new Set();
```

Comment:
optional: instead of putting this at global scope in this module, maybe it's worth putting it in global scope in the page instead and passing it into rectifyCurrentTime and addTimeInput() from the HTML.

Reply:
When you say global scope of the page, do you mean between <script> tags in the HTML page?